.github/test_condarc.yml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - menpo
+  - obspy
```

Comment:
just wanted to check that the obspy and menpo channels are really needed?

Reply:
You are right, we dont need those.